### PR TITLE
Suggested changes to Rationalise _open_resource PR!

### PR DESCRIPTION
**Pull request by andylolz**

### CODE REVIEW

File: iati_datastore/iatilib/console.py [hunk 1]
```diff
@@ -75,30 +75,6 @@ def build_query_builder(deploy_url=None):
...
 
-@click.option(
-        '-x', '--fail-on-xml-errors', "fail_xml")
-@click.option(
-        '-s', '--fail-on-spec-errors', "fail_spec")
-@click.option('-v', '--verbose', "verbose")
-@click.argument('filenames', nargs=-1)
-@cli.command()
-@with_appcontext
-def parse_file(filenames, verbose=False, fail_xml=False, fail_spec=False):
```
Comment (markbrough) [inline iati_datastore/iatilib/console.py:86]: Are we sure this isn't useful? I quite like having command line commands to quickly test particular files etc?

File: iati_datastore/iatilib/parse.py [hunk 2]
```diff
@@ -401,25 +399,6 @@ def default_language(xml, resource=None, major_version='1'):
     return codelists.by_major_version[major_version].Language.from_string(xml_value)
 
 
-def _open_resource_from_file(filename):
```
Comment (markbrough) [inline iati_datastore/iatilib/parse.py:404]: Are we sure this function isn't used elsewhere? I guess not though if tests are passing!

File: iati_datastore/iatilib/console.py (hunk 1 above)
Comment (andylolz) [reply iati_datastore/iatilib/console.py]: Yeah, fair enough. I am not really sure if it is useful tbh. Let’s keep it!

File: iati_datastore/iatilib/parse.py (hunk 2 above)
Comment (andylolz) [reply iati_datastore/iatilib/parse.py:404]: Yep, this is safe to remove.

**Comment by andylolz** [comment]
I’ve amended this PR so that we keep `iati  parse-file`, but just get it working.

**Review by markbrough — APPROVED** [review]
This looks great, thanks a lot!!

**Pull request merged**